language-design.md

```diff
@@ -106,7 +282,29 @@ you do so? If not, why not?
 
 **Response**
 
-
+No, we don't believe that the two experiments disagree with each other. Pavlus 
+is not arguing for a "natural language" but more for a language that is easier 
+to understand by using words from our current language. This is done by changing 
+operators and key words from symbols into words or symbols that are common to 
+our everyday language, like changing "for(int i = 0; i < 10; i++)" to 
+"integer i = 0 \break repeat 10 times \break i = i + 1" [Pavlus, 2012]. Both 
+examples have the exact same functionality, but change the assumed knowledge of 
+for loop construction to words like "repeat" and "times." 
+
+We agree with our previous argument that natural language, if implemented at all, 
+should be implemented in a superficial way. That means that it is only replacing 
+singular functions or key words and not trying to simplify the language. We think 
+the inclusion of natural language into the design of a DSL is dependent on both 
+the intended user and the domain that the DSL covers. If the language of the domain 
+is very well defined and widely used, it would make sense to include parts of it 
+to make understanding of the DSL easier. If the intended user of the language is 
+use to languages that don't include natural language, it would make sense to leave 
+it out of the DSL design. Basically our idea for when not to use it is to think of 
```

Comment:
designing for DSLs is way easier than designing for a GPL because we know (hopefully!) who our target market is!